ansible/playbooks/roles/repository/tasks/RedHat/install-packages.yml

```diff
@@ -21,6 +21,7 @@
       - python36
       - python3-pyyaml
       - rsync  # for Ansible (synchronize module)
+      - tar
```

Comment:
As above, tar is necessary for the download-requirements.py script, which can be called separately, e.g. in offline mode.

Reply:
Fixed in a new commit.